test/webhook.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest'
```

Comment:
this is terrible - none of this test is actually useful.

I'm not sure a test is needed here.